spec/system/reaction_notifications_spec.rb

```diff
@@ -63,4 +63,29 @@
     )
     expect(labels[2]).to have_text(acting_user_1.username)
   end
+
+  context "when prioritize full names in ux site setting is on" do
+    before { SiteSetting.prioritize_full_name_in_ux = true }
+
+    it "renders reaction notifications with full names" do
```

Comment:
Now that we've added the username fallback, I think it's worth adding a test case for when the full name setting is on but users have a null name to test that we actually fallback to the username.